updatem/orchestration/update_orchestrator.go

```diff
@@ -39,11 +40,13 @@ func (orchestrator *updateOrchestrator) Name() string {
 }
 
 // NewUpdateOrchestrator creates a new update orchestrator that does not handle cross-domain dependencies
-func NewUpdateOrchestrator(cfg *config.Config) api.UpdateOrchestrator {
-	return &updateOrchestrator{
-		cfg:          cfg,
-		phaseTimeout: util.ParseDuration("phase-timeout", cfg.PhaseTimeout, 10*time.Minute, 10*time.Minute),
+func NewUpdateOrchestrator(cfg *config.Config, ownerApprovalClient api.OwnerConsentClient) api.UpdateOrchestrator {
+	ua := &updateOrchestrator{
+		cfg:                cfg,
+		phaseTimeout:       util.ParseDuration("phase-timeout", cfg.PhaseTimeout, 10*time.Minute, 10*time.Minute),
```

Comment:
timeout should be configurable, either in this PR or a new issue should be created

Reply:
As of now it is configurable, _cfg.PhaseTimeout_ value is with priority. _cfg.PhaseTimeout_  is configurable through flags and config file. The 10min default values are when _cfg.PhaseTimeout_ is empty or could not be parsed.

Reply:
Or you mean a separate configuration just for owner consent timeout?